app/controllers/api_controller.rb

```diff
@@ -45,6 +45,10 @@ def user_for_paper_trail
     request.headers['X-Current-User']
   end
 
+  def created_by
+    user_for_paper_trail || doorkeeper_application_owner&.name
```

Comment:
are we not going to add `|| 'unknown'` here (noticed this was an option previously?

Reply:
it was an option when the event came from the frontend, it shouldn't be a thing now, or if it is, leaving created_by empty is probably better.